module-apps/apps-common/notifications/NotificationListItem.cpp

```diff
@@ -57,30 +58,35 @@ namespace
         hbox->setMaximumSize(style::window::default_body_width, style::notifications::itemHeight);
     }
 
-    constexpr auto maxNotificationValue = "99+";
-    constexpr auto singleNotification   = "1";
+    constexpr auto singleNotification             = 1;
+    constexpr auto maxNotificationValue           = 99;
+    constexpr auto maxNotificationReplacementText = "99+";
 
-    auto buildNotificationCountText(const UTF8 &indicator) -> gui::Text *
+    auto buildNotificationCountText(unsigned count) -> gui::Text *
```

Comment:
consider using fixed-size type

Reply:
The only place this method is used is in the `NotificationWithEventCounter` constructor, which gets its argument only from `getValue()` method from `NotificationData.cpp`. That method explicitly returns `unsigned` type, so I just used the same type here.